.github/workflows/main.yml

```diff
@@ -385,6 +385,7 @@ jobs:
       matrix:
         workspace:
           - examples/counter
+          - examples/composition
```

Comment:
How about adding something like (**_untested_**):
```suggestion
        include:
          - workspace: examples/counter
             dev-mode: true
          - workspace: examples/composition
             dev-mode: false
```

Reply:
Fixed by running all examples using the local prover.